app/graphql/types/volume.rb

```diff
@@ -0,0 +1,37 @@
+class Types::Volume < Types::BaseObject
+  description 'Multiple Chapters of a Manga'
+
+  field :id, ID, null: false
+
+  field :titles, Types::TitlesList,
+    null: true,
+    description: 'The titles for this volume in various locales'
+
+  def titles
+    {
+      localized: object.titles,
+      canonical: object.canonical_title
+    }
+  end
+
+  field :number, Integer,
+    null: false,
+    description: 'The sequence number of this volume'
+
+  field :thumbnail, Types::Image,
+    null: true,
+    description: 'A thumbnail image for this volume'
+
+  field :chapters_count, Integer,
+    null: false,
+    description: 'Total chapters per volume'
```

Comment:
This makes it sound like a Manga-level attribute vs a volume-level thing.  How about this?

```suggestion
    description: 'Total chapters in this volume'
```